database/1-create-database.sql

```diff
@@ -0,0 +1,34 @@
+
+
+DROP TABLE guest;
+DROP TABLE booking;
+
+use hotel;
+
+CREATE TABLE booking (
```

Comment:
When I tried to create this table in the MySQL console, the console gave me this error, please fix the syntax.

```
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '),
method_payment VARCHAR(20),
PRIMARY KEY(id)
)Engine=InnoDB' at line 6
```